cdc/capture/capture.go

```diff
@@ -359,3 +359,23 @@ func (c *Capture) IsOwner() bool {
 		return nil
 	}) == nil
 }
+
+// GetOwner return the owner of current TiCDC cluster
+func (c *Capture) GetOwner(ctx context.Context) (*model.CaptureInfo, error) {
```

Comment:
https://github.com/pingcap/ticdc/blob/37d3e4b985e7355e27e19641307b8d850d660bcf/cmd/util.go#L118 

A similar method.

Reply:
I don't know what to do with it at the moment. Maybe we can add a todo and find a way to clean up the duplicate code.

Reply:
Yeh, we can clean up the code in `/cmd/util` in the future. (i will do it)
I think it is better that cli can not access etcd_client directly.